charms/jupyter-ui/tests/integration/test_charm.py

```diff
@@ -68,3 +69,41 @@ async def test_ui_is_accessible(ops_test: OpsTest):
     assert result_status == 200
     assert len(result_text) > 0
     assert "Jupyter Management UI" in result_text
+
+
+@pytest_asyncio.fixture
+async def deploy_kubeflow_dashboard(ops_test: OpsTest):
+    """Deploys kubeflow-dashboard and kubeflow-profiles, cleaning them up after the usage."""
+    kubeflow_profiles = "kubeflow-profiles"
+    await ops_test.model.deploy(kubeflow_profiles, channel="stable", trust=True)
```

Comment:
Why stable?

Reply:
changed to edge because stable was not up to date